client/landing/gutenboarding/onboarding-block/design-selector/index.tsx

```diff
@@ -152,14 +149,14 @@ const DesignSelector: FunctionComponent = () => {
 
 			<Portal>
 				<DialogBackdrop
-					visible={ hasSelectedDesign }
+					visible={ isPageSelectorOpen }
 					className="design-selector__page-layout-backdrop"
 				/>
 			</Portal>
 
 			<Dialog
 				{ ...dialog }
-				hide={ resetState }
+				hide={ () => setIsPageSelectorOpen( false ) }
```

Comment:
With this change, when the user picks one design, the selection is persisted even after closing the Page Selector, so the `Create my site` button stay visible linking to the last selected design.